docs/setup.asciidoc

```diff
@@ -0,0 +1,174 @@
+[[setup]]
+== Get started
+
+[float]
+[[setup-step-1]]
+=== Step 1: Install
+
+Add the package to your `go.mod` file:
+
+[source,go]
+----
+require go.elastic.co/ecszap master
+----
+
+[float]
+[[setup-step-2]]
+=== Step 2: Configure
+
+Set up a default logger. For example:
+
+[source,go]
+----
+encoderConfig := ecszap.NewDefaultEncoderConfig()
+core := ecszap.NewCore(encoderConfig, os.Stdout, zap.DebugLevel)
+logger := zap.New(core, zap.AddCaller())
+----
+
+[float]
+[[examples]]
+=== Examples
+
+[float]
+[[use-structured-logging]]
+==== Use structured logging
+
+[source,go]
+----
+// Add fields and a logger name
+logger = logger.With(zap.String("custom", "foo"))
+logger = logger.Named("mylogger")
+
+// Use strongly typed Field values
+logger.Info("some logging info"),
+    zap.Int("count", 17),
+    zap.Error(errors.New("boom")),
+}
```

Comment:
Can you look at this example? I think the readme was missing a closing `)`. I added it in after "some logging info", but I'm not sure if it's in the correct place.

<img width="277" alt="Screen Shot 2021-01-14 at 4 05 26 PM" src="https://user-images.githubusercontent.com/5618806/104663677-5b27bc00-5682-11eb-9b04-86603b2cf357.png">